turbinia/processors/mount_local.py

```diff
@@ -51,15 +51,35 @@ def PreprocessMountDisk(evidence):
 
   evidence.mount_path = tempfile.mkdtemp(prefix='turbinia', dir=mount_prefix)
 
-  if hasattr(evidence, 'mount_partition') and evidence.mount_partition:
-    src_path = '{0:s}-part{1:d}'.format(
```

Comment:
We should find a way to handle the cloud device path as well...

Reply:
The function here will have  something like  `/dev/disk/by-id/google-DISKNAME-embedded` passed as evidence.local_path, and losetup will be quite happy and still return a /dev/loopX, where /dev/loopXp1 is the same as dev/disk/by-id/google-DISKNAME-embedded-part1

Reply:
I was able to run `turbiniactl googleclouddiskembedded` with the code above (and the commits below :) )